CHANGES.rst

```diff
@@ -4,6 +4,9 @@ History
 0.2.1 (unreleased)
 ------------------
 
+- Add configure command
```

Comment:
Please do meaningful chnage notes.
What is the name of the new command and what does it do?